airmozilla/starred/static/starred/css/star_event.css

```diff
@@ -0,0 +1,18 @@
+.star {
+    background: url('../img/Star24YellowTransparent.png') no-repeat;
+    height: 24px;
+    width: 24px;
+    background-position: 0 0;
+    position: absolute;
+    left: 140px;
+    top: 70px;
+    opacity: 0.9;
```

Comment:
At the moment we have, in the HTML, this: `<a name="star star-off"></a>`  which, due to the lack of a `href` attribute, does get the pointer cursor when you over the mouse over it. So add to this CSS:
```css
cursor: pointer;
```